### ISSUE TITLE
Fastly CLI reports error when using is using ARM macOS

### ISSUE THREAD

**Issue by Integralist** [fastly/Viceroy#87]
✋🏻 Hello

We've had [a report](https://github.com/fastly/cli/issues/458) from @yusukebe that the Fastly CLI is displaying an error to say that a corresponding Viceroy asset cannot be found:

```
ERROR: error downloading latest Viceroy release: no asset found for your OS (darwin) and architecture (arm64).
```

The error is accurate because there is indeed no ARM asset produced for Viceroy.

I realised there was a prior conversation regarding ARM support, see comments from [the Windows support PR](https://github.com/fastly/Viceroy/pull/82#issuecomment-947779171):

```
There are dependencies on C and assembly code (ex: wasmtime-fiber), 
so cross-compilation is not going to work unless you install an additional toolchain.

Long story short, we have to use qemu in order to build the ARM packages.
```
